templates/frag/fromMultipart.hbs

```diff
@@ -0,0 +1,38 @@
+{{!--
+Output a reference to a function to transform the multipart input value to a JavaScript value.
+@param <this> CodegenSchema
+@param prefix the import prefix for the functions
+--}}
+{{#if (isBoolean)}}
+{{prefix}}booleanFromJson
+{{~else if (isInteger)~}}
+{{prefix}}integerFromJson
+{{~else if (isNumber)~}}
+{{prefix}}numberFromJson
+{{~else if (isString)~}}
+{{prefix}}stringFromJson
+{{~else if (isBinary)~}}
+{{prefix}}binaryFromJson
+{{~else if (isFile)~}}
+{{prefix}}fileFromMultipart
+{{~else if (isObject)~}}
+{{prefix}}model{{className nativeType.parentType}}FromMultipart
```

Comment:
OOOOOOH! I think we can drop the whole "FromMultipart" and "FromJson" thing from our naming. We generate _either_ the multipart version _or_ the JSON version for a given schema. Let's not change that now but as soon as we merge this... let's try to replace "FromJson" and "FromMultipart" with "FromValue".